Assets/Plugins/Source/Editor/Configs/SteamConfig.cs

```diff
@@ -25,23 +25,61 @@
 
 namespace PlayEveryWare.EpicOnlineServices
 {
+    using Editor;
+    using Editor.Config;
+    using Editor.Utility;
+
     [Serializable]
+    [ConfigGroup("Steam Configuration")]
+    // TODO: Make SteamConfig derive from EditorConfig, and update the native code
+    //       to properly reference the correct file where appropriate.
     public class SteamConfig : Config
     {
+        [ConfigField("Steam Flags", ConfigFieldType.TextList)]
+        [Obsolete] // This is marked as obsolete because it is actually not utilized anywhere.
         public List<string> flags;
+
+        #region These fields are referenced by the native code 
+
+        [DirectoryPathField("Override Library Path")]
         public string overrideLibraryPath;
 
+        [ConfigField("Steamworks SDK Major Version", ConfigFieldType.Uint)]
         public uint steamSDKMajorVersion;
+
+        [ConfigField("Steamworks SDK Minor Version", ConfigFieldType.Uint)]
         public uint steamSDKMinorVersion;
 
+        [ConfigField("Steamworks Interface Versions", ConfigFieldType.TextList)]
         public List<string> steamApiInterfaceVersionsArray;
 
+        #endregion
+
+        [ButtonField("Update from Steamworks.NET")]
+        public Action UpdateFromSteamworksNET;
```

Comment:
Can this be assigned to a function, or do we have to do this pattern of setting the Action in the constructor?

Reply:
See my [note](https://github.com/PlayEveryWare/eos_plugin_for_unity/pull/822#discussion_r1701052663) on your other comment, I think it helps explain how this works and why it works the way that it does.